src/index.tsx

```diff
@@ -12,15 +12,15 @@ import {
 import './index.css';
 import App from './modules/app/pages/App';
 import * as serviceWorker from './utils/serviceWorker';
-import {
-    blue,
-    orange
-} from '@material-ui/core/colors';
 
 const theme: Theme = createMuiTheme({
     palette: {
-        primary: blue,
-        secondary: orange,
+        primary: {
```

Comment:
Can we use colors that are not so harsh on the eyes. We should be using the material color pallete or at least softer versions of these colors.

I suggest we use material color pallete because the color schemes are backed with accessibility research. Yes, it does mean that we stray away from specific CG colors, but these shades don't work well for a web app

Reply:
https://material.io/design/color/

Reply:
So when I talked to Sandy about it, she rathered the consistency. We need to talk to the Campus Staff about it and figure out what they want. I would rather keep it consistent just so it is clear to our public users that this is our site.

Reply:
So I will have to talk with her about accessibility but for now keep it out of the PR

Reply:
actually, I tested the colors and they are accessible

Reply:
keep them in

Reply:
ACTUALLY (sorry for flip flopping) I did more accessibility tests and these colors don't pass some key tests. We need to keep this out and we can all talk together later so I can show you whats going wrong

Reply:
I will take them out for now, but I think this a conversation for the staff. If these are not accessible, then our entire current web page is most definitely inaccessible. I am not sure what is more important: staying consistent or not making the same mistake again.